src/pages/preferences/preferencesTabs/NewUITab.tsx

```diff
@@ -50,18 +52,40 @@ export const NewUITab: React.VFC = () => {
       refetchQueries: ["GetUserSettings"],
     });
   };
+
+  const handleOnChangePolling = () => {
+    const nextState = Cookies.get(DISABLE_QUERY_POLLING) !== "true";
+    sendEvent({
+      name: "Toggle polling",
+      value: nextState ? "Enabled" : "Disabled",
+    });
+    Cookies.set(DISABLE_QUERY_POLLING, nextState.toString());
+    window.location.reload();
+  };
+
   return (
     <>
       {/* @ts-expect-error */}
       <PreferencesCard>
         <PaddedBody>
           Direct all inbound links to the new Evergreen UI, whenever possible
-          (e.g. from the CLI, GitHub, etc.)
+          (e.g. from the CLI, GitHub, etc.).
         </PaddedBody>
         <Toggle
           checked={spruceV1}
           disabled={updateLoading}
-          onChange={handleToggle}
+          onChange={handleOnChangeNewUI}
+          aria-label="Toggle new evergreen ui"
+        />
+      </PreferencesCard>
+      {/* @ts-expect-error */}
+      <PreferencesCard>
+        <PaddedBody>
+          Allow background polling for the current browser.
```

Comment:
Should we add a description for how polling works on spruce. (Explain the thing about polling being disabled when you change tabs) I can see that being confusing for users and it has come up on #evergreen-users before.